### PR TITLE
Add PHP example

### LINKED ISSUE #42: Need PHP Example

### PR DESCRIPTION
**Pull request by mehedimi**
This PR will fix #42 issue

### CODE REVIEW

File: readme.MD [hunk 1]
```diff
@@ -174,3 +174,37 @@ createTaskRequest := taskspb.CreateTaskRequest{
 }
 createdTaskResp, _ := client.CreateTask(context.Background(), &createTaskRequest)
 ```
+
+### PHP example
+PHP example looks like this.
+```php
+<?php
```
Comment (aertje) [inline readme.MD:181]: Think this can be left out as it's implicit?

Comment (mehedimi) [reply readme.MD:181]: Thanks for pointing it out. I have updated that text.

Comment (aertje) [reply readme.MD:181]: Oh sorry for not being clearer in my comment: I meant the `<?php` opening tag is superfluous.

Comment (mehedimi) [reply readme.MD]: OK I have removed the `<?php` tag.

**Pull request merged**